nautobot/dcim/tables/template_code.py

```diff
@@ -85,7 +85,7 @@
 
 UTILIZATION_GRAPH = """
 {% load helpers %}
-{% utilization_graph value %}
+{% utilization_graph value.0 value.1 %}
```

Comment:
Consider using something like a `dict` or a `namedtuple` for the return value from `get_utilization` and `get_power_utilization` so that we don't need to use magic indices like this.

Reply:
Yes, using a named tuple here would allow for more readable code. 

e.g. 

```
{% utilization_graph value.occupied_unit_count value.u_height %}
```